src/main/java/step3/domain/Height.java

```diff
@@ -0,0 +1,37 @@
+package step3.domain;
+
+import java.util.Objects;
+
+public class Height {
+    public static final int MINIMUM_HEIGHT = 2;
+    private static final String MINIMUM_HEIGHT_ERROR = "사다리의 높이는 최소 2 이상이어야 합니다.";
+    private final int height;
+
+    public Height(int height) {
+        checkMinimumHeight(height);
+        this.height = height;
+    }
+
+    public int height() {
+        return height;
+    }
```

Comment:
```suggestion
    public int getValue() {
        return height;
    }
```

getter를 사용하지 않고 하는 방법이 어려우시다면
getter를 적용해서 쓰시면 좋을 것 같아요 ~ :)